### DISCUSSION TITLE
Ideas on how to leverage OpenAI's Generative Artificial Intelligence platform

### DISCUSSION THREAD

**Discussion by mvilanova** [Netflix/dispatch#3174] [Ideas]
This week we merged a [plugin](https://github.com/Netflix/dispatch/pull/3159) for OpenAI's Generative Artificial Intelligence platform and we have started brainstorming internally how we could potentially leverage OpenAI to augment Dispatch's Case and Incident Management's capabilities. You can find some initial ideas below. I'd be interested in hearing from others in the community as well. Thanks!

* Incident summarization in Layman terms
* Incident timeline generation
* Recommendations based on metadata and existing incident types and tagging ontology for incident:
  * Title
  * Description
  * Resolution
  * Types
  * Tags
* Metrics and insights reporting

**Comment by vikassinha2019** [comment]
It should be able to generate Incident summarization on the basis of slack chat and zoom audio transcript.
It should help with the content for posting statuses on Status page. If it can generate the content on the basis of Incident summarization, it would be great.

**Comment by sara4dev** [comment]
Also answering analytics questions like "which component is causing major incidents?", what is the average MTTR?, etc.

Here's an example of chatgpt's analytics feature explained - https://twitter.com/petergyang/status/1655580839151906818?s=20
